src/config.ts

```diff
@@ -166,6 +170,11 @@ const buildAppConfig = async (secrets: any): Promise<AppConfig> => {
       policy,
       writeScope: `${policy}.WRITE`,
       readScope: `${policy}.READ`,
+      egoRootRest: process.env.EGO_ROOT_REST || 'http://localhost:8081',
+      egoScoreProxyClientId:
```

Comment:
I followed the same pattern as a few other Vault-protected secrets for this. Haven't tested the Vault stuff yet, just been setting these in my .env

Reply:
same change for the ENV file variables as well...